tests/Integration/Infrastructure/Controller/Regulation/AddRegulationControllerTest.php

```diff
@@ -61,6 +65,8 @@ public function testBadPeriod(): void
         $form['general_info_form[description]'] = 'Travaux';
         $form['general_info_form[startDate]'] = '2023-02-12';
         $form['general_info_form[endDate]'] = '2023-02-11';
+        $form['general_info_form[category]'] = RegulationOrderCategoryEnum::ROAD_MAINTENANCE->value;
```

Comment:
(Pinaillage de l'espace) Respecter l'ordre des champs dans le formulaire ?

```suggestion
        $form['general_info_form[category]'] = RegulationOrderCategoryEnum::ROAD_MAINTENANCE->value;
        $form['general_info_form[description]'] = 'Travaux';
        $form['general_info_form[startDate]'] = '2023-02-12';
        $form['general_info_form[endDate]'] = '2023-02-11';
```